cypress/integration/users/delete-users.spec.js

```diff
@@ -0,0 +1,72 @@
+import {
+  Button,
+  including,
+  Pane,
+} from '../../../interactors';
+
+describe('Deleting user', () => {
+  const lastName = 'Test123' + Number(new Date()).toString();
+  const ResultsPane = Pane({ index: 2 });
+  const ActionsMenu = ResultsPane.find(Button({ className: including('actionMenuToggle') }));
+  const DeleteButton = Button({ id: 'clickable-checkdeleteuser' });
+  const ModalButtonYes = Button({ id: 'delete-user-button' });
+  const ModalButtonNo = Button({ id: 'close-delete-user-button' });
+
+  before(() => {
+    cy.login('diku_admin', 'admin');
+    cy.getToken('diku_admin', 'admin');
+  });
+
+  beforeEach(() => {
+    cy.getUserGroups({ limit: 1 }).then(() => {
+      const userData = {
+        active: true,
+        personal: {
+          preferredContactTypeId: '002',
+          lastName,
+          email: 'test@folio.org',
+        },
+        patronGroup: Cypress.env('userGroups')[0].id,
+        departments: []
+      };
+      cy.postUser(userData);
+    });
+  });
+
+  afterEach(() => {
+    cy.getUsers({ query: `personal.lastName="${lastName}"` })
+      .then(() => {
+        Cypress.env('users').forEach(user => {
+          cy.deleteUser(user.id);
+        });
+      });
+  });
+
+  it('should be possible by user delete action', function () {
+    const { id } = Cypress.env('user');
+
+    cy.visit(`/users/preview/${id}`);
+    cy.do([
+      ActionsMenu.click(),
```

Comment:
it can be common command open menu and press action

Reply:
I can not imagine how this command could look like. Should it get the action as an argument?